packages/d3fc-webgl/README.md

```diff
@@ -2,15 +2,403 @@
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aBandwidth` shader attribute.
+
+<a name="webglSeriesOhlc_definedAttribute" href="#webglSeriesOhlc_definedAttribute">#</a> *webglSeriesOhlc*.**definedAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aDefined` shader attribute.
+
+Values calculated from the given *attributeBufferBuilder* should either be `1` (to indicate the data point is defined) or `0` (to indicate the data point is not defined).
+
+#### Point
+
+<a name="webglSeriesPoint" href="#webglSeriesPoint">#</a> fc.**webglSeriesPoint**()
+
+Used to construct a new WebGL Point series.
+
+<a name="webglSeriesPoint_type" href="#webglSeriesPoint_type">#</a> *webglSeriesPoint*.**type**(*symbolTypeShader*)
+
+If *symbolTypeShader* is specified, sets the symbol type shader and returns this series. If *symbolTypeShader* is not specified, returns the current symbol type shader.
+
+A *symbolTypeShader* can be generated using the [webglSymbolMapper](Link to webglSymbolMapper).
+
+<a name="webglSeriesPoint_crossValueAttribute" href="#webglSeriesPoint_crossValueAttribute">#</a> *webglSeriesPoint*.**crossValueAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aCrossValue` shader attribute.
+
+<a name="webglSeriesPoint_mainValueAttribute" href="#webglSeriesPoint_mainValueAttribute">#</a> *webglSeriesPoint*.**mainValueAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aMainValue` shader attribute.
+
+<a name="webglSeriesPoint_sizeAttribute" href="#webglSeriesPoint_sizeAttribute">#</a> *webglSeriesPoint*.**sizeAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aSize` shader attribute.
+
+<a name="webglSeriesPoint_definedAttribute" href="#webglSeriesPoint_definedAttribute">#</a> *webglSeriesPoint*.**definedAttribute**(*attributeBufferBuilder*)
+
+Sets the *attributeBufferBuilder* to be used to create the buffer for the `aDefined` shader attribute.
+
+Values calculated from the given *attributeBufferBuilder* should either be `1` (to indicate the data point is defined) or `0` (to indicate the data point is not defined).
+
+### Attribute Buffer Builders
+
+The attribute components can be used to generate buffers for a series and bind them to a given program context. A number of different builders are available to accomodate the most common use cases.
+
+#### Element Attribute
+
+<a name="webglElementAttribute" href="#webglElementAttribute">#</a> fc.**webglElementAttribute**()
+
+Used to generate a buffer containing values to be used on a per component basis. In this context a component is each object drawn to a chart, for example each individual candlestick on a candlestick chart is a component.
```

Comment:
We need to be crazy pedantic about using the correct domain terminology, otherwise it makes it much harder for the reader to follow -

Used to generate a buffer containing values to be used on a per element basis. In this context an element is an instance of a repeatedly drawn object, e.g. each individual candlestick on a candlestick chart is an element.